pkg/cmd/openshift-integrated-oauth-server/cmd.go

```diff
@@ -1,12 +1,12 @@
-package openshift_osinserver
+package openshift_integrated_oauth_server
```

Comment:
nit: the long package name that doesn't match the directory name 😢

Reply:
`pkg/cmd/openshift-integrated-oauth-server` is consistent with the other `pkg/cmd` binaries and explicit about what the process is.  Dashes are not allowed in package names.